modules/users/client/views/authentication/authentication.client.view.html

```diff
@@ -1,24 +1,12 @@
-<section class="row">
+<section class="row" data-ng-controller="AuthenticationController">
 	<h3 class="col-md-12 text-center">Sign in using your social accounts</h3>
 	<div class="col-md-12 text-center">
-		<a href="/api/auth/facebook" target="_self" class="undecorated-link">
-			<img src="/modules/users/img/buttons/facebook.png">
-		</a>
-		<a href="/api/auth/twitter" target="_self" class="undecorated-link">
-			<img src="/modules/users/img/buttons/twitter.png">
-		</a>
-		<a href="/api/auth/google" target="_self" class="undecorated-link">
-			<img src="/modules/users/img/buttons/google.png">
-		</a>
-		<a href="/api/auth/linkedin" target="_self" class="undecorated-link">
-			<img src="/modules/users/img/buttons/linkedin.png">
-		</a>
-		<a href="/api/auth/github" target="_self" class="undecorated-link">
-			<img src="/modules/users/img/buttons/github.png">
-		</a>
-		<a href="/api/auth/paypal" target="_self" class="undecorated-link">
-			<img src="/modules/users/img/buttons/paypal.png">
-		</a>
+		<img ng-click="callOauthProvider('/api/auth/facebook')" src="/modules/users/img/buttons/facebook.png">
```

Comment:
@ilanbiala let me know once you think the modifications are ready, so that I can squash the commits